app/src/main/res/layout/settings_toolbar.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<androidx.appcompat.widget.Toolbar
+    android:id="@+id/toolbar"
+    android:layout_width="match_parent"
+    android:layout_height="?attr/actionBarSize"
+    android:theme="@style/ThemeOverlay.AppCompat.ActionBar"
+    app:titleTextColor="@android:color/white"
```

Comment:
is it meaningful to set the titleTextColor and background explicitly? Isn't that covered by proper theme configuration?

Reply:
Obsolete, removed whole file